airbyte-integrations/connectors/source-sendgrid/source_sendgrid/client.py

```diff
@@ -0,0 +1,57 @@
+"""
+MIT License
+
+Copyright (c) 2020 Airbyte
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in all
+copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
+"""
+
+import json
+import pkgutil
+
+from airbyte_protocol import AirbyteStream
+from python_http_client import ForbiddenError, UnauthorizedError
+from sendgrid import SendGridAPIClient
+
+
+class Client:
+    ENTITIES = ["campaigns", "lists"]
```

Comment:
Could we centralize declaration of recognized entities? For example, here we duplicate defining the entities: https://github.com/airbytehq/airbyte/pull/1002/files#diff-53b9c7a3d7080ac5abf192cc80471b8aea8d5e9d635e77b6d69a3c089aa324d7R78

can we instead declare and define entities in once place? maybe just replace this with the entity mapping and expose `ENTITIES = Object.keys(mapping)` ?